clio-server/src/test/scala/org/broadinstitute/clio/server/webservice/WgsUbamWebServiceSpec.scala

```diff
@@ -133,10 +158,18 @@ class WgsUbamWebServiceSpec
     // We have to test the MemorySearchDAO because we're not going to implement
     // Elasticsearch logic in our test specs. Here, we're just verifying that
     // the web service passes the appropriate queries onto the search DAO.
-    Post("/query", Map("flowcell_barcode" -> "FC123")) ~> webService.queryall ~> check {
-      memorySearchDAO.queryWgsUbamCalls should have length 1
-      val secondQuery = memorySearchDAO.queryWgsUbamCalls.head
-      secondQuery.flowcellBarcode should be(Some("FC123"))
+    Post("/queryall", Map("flowcell_barcode" -> "FC123")) ~> webService.queryall ~> check {
+      memorySearchDAO.queryCalls should be(
+        Seq(
+          // From /query call earlier in the test.
+          TransferWgsUbamV1QueryInput(
+            flowcellBarcode = Some("FC123"),
+            documentStatus = Some(DocumentStatus.Normal)
+          ),
+          // No documentStatus restriction from /queryall
+          TransferWgsUbamV1QueryInput(flowcellBarcode = Some("FC123"))
```

Comment:
When I added a status check to the prior version of the the test, the test failed with `Request was rejected`.

Reply:
That'd be from the route mismatch in `Post("/query") ~> webService.queryall`.

Reply:
Yeah, but for the record, even just reporting the status provoked a rejection.
(Maybe check doesn't call awaitResult unless something in the body calls for it?)